gradle-plugin/src/test/groovy/helper/BaseSpec.groovy

```diff
@@ -39,7 +41,11 @@ repositories { jcenter() }
     }
 
     AkkaGrpcPluginExtension sampleSetup(def plugin = "scala", String scala = "2.12") {
-        project.pluginManager.apply plugin
+        if (plugin == "scala" || plugin == ScalaWrapperPlugin) {
+            def scalaDir = projectDir.newFolder('src', 'main', 'scala')
+            new File(scalaDir, "test.scala").text = "object AkkaGrpc"
```

Comment:
Interesting! I guess it'd be nice to flesh this out further (perhaps somehow include the quickstart example?), but that's something for the future

Reply:
Excuse me , I didn't understand your intentions about this.

Reply:
Sorry, I misread this fragment - nothing needs to change here for this PR ;).